CSS for pupup when slideshow image has link

dkhasham opened the pull request:
Related Ticket: https://synjira.atlassian.net/browse/CELDEV-241

Commit:
CSS for pupup when slideshow image has link

fpichler commented:
slideshow css must go into the celements-photo project. Please move your changes to the Gallery.css file: https://github.com/celements/celements-photo/blob/dev/web-module/src/main/webapp/resources/celRes/Gallery.css

Commit:
min width of contextmenu container

Commit:
moved css for slideshow contextmenu to gallery.css in celements-photo

dkhasham commented:
moved css for contextmenu slideshow to celements-photo project images/slideshow.css. there is no need for this branch anymore.
Closed PullRequest and deleted Branch on git
